documentation/src/main/asciidoc/topics/proc_server_using_datasources.adoc

```diff
@@ -0,0 +1,36 @@
+[id='using_datasources']
```

Comment:
I broke this into a separate procedure because it's different to the reference content. We might also reuse this in the configuration guide for JDBC cache store config.